test/functional/wallet_disable.py

```diff
@@ -26,10 +26,6 @@ def run_test (self):
         x = self.nodes[0].validateaddress('mneYUmWYsuk7kySiURxCi3AGxrAqZxLgPZ')
         assert x['isvalid'] == True
 
-        # Checking mining to an address without a wallet. Generating to a valid address should succeed
-        # but generating to an invalid address will fail.
```

Comment:
Review note - this comment has been removed. Imo it's fine since all of the `generate` tests are with wallet disabled, so the comment doesn't add much value.